zou/app/services/shots_service.py

```diff
@@ -1060,13 +1059,17 @@ def get_shot_versions(shot_id):
 
 
 def get_base_entity_type_name(entity_dict):
-    type_name = "Asset"
```

Comment:
Can you let the variable as declared first?

Reply:
Yes, it's done.